Add missing closing parentheses

gustavwilliam opened the pull request:
A closing parentheses was missed when nesting two parentheses. The following highlights the change made:
```diff
- (text (text)
+ (text (text))
```

gustavwilliam committed:
Add missing closing parentheses

A closing parentheses was missed when nesting two parentheses, like this: (text (text), instead of (text (text)).

hamishwillee commented on files/en-us/web/javascript/guide/details_of_the_object_model/index.html:
```diff
@@ -395,7 +395,7 @@ <h2 id="More_flexible_constructors">More flexible constructors</h2>
 
 <div class="note">
-<p><strong>Note:</strong> This may not work as expected if the constructor function is called with arguments which convert to <code><code>false</code></code> (like <code>0</code> (zero) and empty string (<code><code>""</code></code>). In this case the default value will be chosen.</p>
+<p><strong>Note:</strong> This may not work as expected if the constructor function is called with arguments which convert to <code><code>false</code></code> (like <code>0</code> (zero) and empty string (<code><code>""</code></code>)). In this case the default value will be chosen.</p>
```
Thanks! Perhaps also remove the doubled up "code" - e.g. `<code><code>false</code></code>` and `<code><code>""</code></code>` ?

gustavwilliam committed:
Remove unnecessary nesting of code blocks

Some code blocks were surrounded with redundant code block markup. This has been fixed, like in the following example:

Old: <code><code>false</code></code>
New:  <code>false</code>

gustavwilliam commented:
Thanks for pointing that out @hamishwillee! I've fixed all occurrences of redundant code block nesting in the file.

gustavwilliam committed:
Merge branch 'main' into patch-1

sideshowbarker commented:
Thanks much, and congrats on landing your first patch here — welcome aboard 🎉
